staging/utils/errorx/error.go

```diff
@@ -229,10 +223,8 @@ func (e HttpError) Error() string {
 			}
```

Comment:
Before will print error with http status while ev.IsNil, after removing the status, this switch case is confused and may return an empty string, which needs to be removed or updated. Could refer to `ErrorX’s Error()`